ompi/attribute/attribute_predefined.c

```diff
@@ -194,11 +191,12 @@ static int create_comm(int target_keyval, bool want_inherit)
     ompi_attribute_fn_ptr_union_t del;
 
     keyval = -1;
-    copy.attr_communicator_copy_fn = (MPI_Comm_internal_copy_attr_function*)
-        (want_inherit ? MPI_COMM_DUP_FN : MPI_COMM_NULL_COPY_FN);
+    copy.attr_communicator_copy_fn = (MPI_Comm_internal_copy_attr_function
```

Comment:
no!

Reply:
Oh, I completely agree. Working on this one now. Looking for other places where the formatting is weird. Generally these sorts of things come up during the initial formatting but get resolved as the code evolves. They should have a penalty for breaking in a type like this. Would definitely set it to 10000.